### ISSUE TITLE
[BUG] Low resolution on Linux

### ISSUE THREAD

**Issue by Nehctarg** [jeertmans/manim-slides#58]
### Description

I've run following commands:
```
$ manim render example.py ThreeDExample
$ manim-slides ThreeDExample # This results in a low quality animation, but with slides
$ mpv "$(pwd)"/media/videos/main/2160p60/ThreeDExample.mp4 # This results in a high quality animation
```
This leads me to believe that manim-slides converts the mp4 files in a way that leads to a reduction of quality. 
 

### Version

manim-slides, version 4.2.0

### Platform

Linux, version 6.0.2-arch1-1

### Screenshots

Look at the arrows of the coordinate system, the  arrow in the video is clearly sharper. 
Using `manim-slides ThreeDExample`:
![1666504743](https://user-images.githubusercontent.com/110668837/197376553-8022a03c-f5fe-469b-9ec7-cee9758a6121.png)

Using ` mpv "$(pwd)"/media/videos/main/2160p60/ThreeDExample.mp4`:
![1666504244](https://user-images.githubusercontent.com/110668837/197376359-6e4211b7-4929-4af6-a9a9-350388c43ca9.png)

### Additional information

Python version: 3.10.8
Qt specific environment variables:
- QT_QPA_PLATFORM=wayland
- QT_QPA_PLATFORMTHEME=qt5ct 

wayland compositor: 
river, compiled without xwayland support

**Comment by Nehctarg** [comment]
This is quite similar to #20, but with a different platform.
Reducing the screen scaling is not helping either.

**Comment by jeertmans** [comment]
Hi @Nehctarg! Thank you for raising this issue!

When using no resizing, rendering in a specific resolution and using the same resolution flag with manim-slides, does the image look the same?

**Comment by Nehctarg** [comment]
I've ran 
```bash
manim render -q m example.py ThreeDExample # m stands for 1280x720 30FPS
manim-slides present --resolution 1280 720 ThreeDExample
```
And the result is, although now at a lower resolution, still not sharp.

**Comment by jeertmans** [comment]
After looking at this problem, I cannot observe a "lower" resolution with Manim. I'm not using Wayland (unfortunately), but don't you think the resolution difference is due to use a video player that (most probably) uses anti-aliasing or some smoothing technique?

When comparing between VLC and Manim Slides, I first see that VLC does not display my content in 1280 x 720, even though I specified to not rescale. So the comparison is therefore biased.

When rescaling both Windows to be as close as possible, this is what I observe:
![manim](https://user-images.githubusercontent.com/27275099/197462300-a39ab6fc-8cfc-429a-bf67-ace9ffce78ce.png)

![vlc](https://user-images.githubusercontent.com/27275099/197462216-160ca73b-6250-41ba-a8da-60238b042faa.png)

If having a smooth rescaling is important to you, I have made a PR #59 that will soon be merged and released.
